external/libtww/include/d/com/d_com_inf_game.h

```diff
@@ -1608,6 +1608,8 @@ inline u8 dComIfGs_getChartDeciphered() {
     return g_dComIfG_gameInfo.info.getPlayer().mMap.getChartDeciphered();
 }
 
+LIBTWW_DEFINE_FUNC(dComIfGs_setGameStartStage__Fv,void, setGameStartStage, (void));
+
```

Comment:
In this case it would be good to just make the function name the real function name from the game, so `dComIfGs_setGameStartStage`

Reply:
Made the correction. Thanks.